Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM gliderlabs/alpine
+
+COPY requirements.txt /opt/elodie/requirements.txt
+COPY docs/requirements.txt /opt/elodie/docs/requirements.txt
+COPY elodie/tests/requirements.txt /opt/elodie/elodie/tests/requirements.txt
+WORKDIR /opt/elodie
+
+RUN apk add --update boost-python ca-certificates exiftool exiv2 make python py-pip && \
```

Comment:
Aha this explains why it did not work on my side (for another project)! Nice catch for LatLon btw.